1 table.sql

```diff
@@ -0,0 +1,31 @@
+use TenantDB
+go
+
+drop table if exists Tenant
+go
+create table dbo.Tenant(
+    TenantId int identity primary key,
+    TenantName varchar(50) not null constraint ck_Tenant_TenantName_cannot_be_blank check(TenantName > ''),
+    ContactName varchar(50) not null constraint ck_Tenant_ContactName_cannot_be_blank check(ContactName > ''),
+    ContactEmail varchar(50) null constraint ck_Tenant_ContactEmail_cannot_be_blank check(ContactEmail > ''),
+                             constraint ck_Tenant_ContactEmail_must_be_valid_email check(ContactEmail like '%@%.%'),
+    ContactPhone varchar(12) null constraint ck_Tenant_ContactPhone_cannot_be_blank check(ContactPhone > ''),
+    UnitNumber int not null constraint ck_Tenant_UnitNumber_must_be_between_1_and_20 check(UnitNumber between 1 and 20),
```

Comment:
Yes, but then all the rest of the data would have to be together - like the unit square footage and all the pricing etc.  Do you still want me to change the table to allow 2 units under one lease record?

Reply:
Yes, the only part that would really be different is the Unit number. the square footage, pricing would just be larger, and the table as it is now allows for that.

Reply:
Really, this scenario would be better structured by using multiple tables.  But since at this point in the course, the instructions for this table were to create only one table, that's how I did it.   With one table, the way to put 2 units under one lease would be to make the UnitNumber column a varchar, instead of an int (since if there are more than one unit number, they will need to be written out with commas).  Therefore, I think I'll just leave it as is.  If the unit number column is a varchar, then it will be hard to produce accurate reports.